content/departments/product-engineering/engineering/code-graph/search/guiding_principles.md

```diff
@@ -0,0 +1,41 @@
+
+Our job as you onboard is to give you this context, so that you can liberate your ability. As a result, your job as you onboard is to focus on absorbing context, rather than attempting to demonstrate ability through immediate execution. This doesn't mean operating solely in the abstract, as of course part of absorbing context will entail tackling some practical problems.
+
+## We'll provide direction, not tasks
+
+While we will provide you with some initial tasks that will help you absorb context, the eventual end goal is that you are able to identify the work you should tackle to produce impact towards team and company goals. This impact can be direct or indirect. An example of direct impact is building functionality that will directly unblock a user outcome. An example of indirect impact is ensuring we have a solid foundation to build upon, so that we can build functionality that unblocks user outcomes.
+
+Our job is to help provide context and clarity on direction: what problems are we seeking to solve as a team, as a company? Why do these problems matter? What are north star outcomes? As you absorb this direction, and understand how it relates to your domain of expertise, you will be empowered to directly identify and break down the work that should happen to make progress towards north star outcomes. As you wrap up a work item, you should feel empowered to ask yourself: "what should I work on next? why?", and to bring your answers to the team. In your first few months, you may find that we end up asking you these questions ("what do _you_ think you should work on next?"), to encourage this practice.
+
+## Your scope can be arbitrarily wide
+
+While you may naturally gravitate towards the types of problems you have the most expertise solving, and while we will pick initial tasks that match your expertise, you do not need special permission to tackle problems outside of this scope. A practical example could be a backend-leaning engineer tackling frontend-leaning tasks. However, scope extends beyond the technical realm here: you should feel empowered to, for instance, suggest improvements to our team processes, challenge or clarify our team strategy, or bring forward opportunities to collaborate with other teams.
+
+We trust you to leverage this freedom with discernment, keeping good forward momentum towards the outcomes you, the team and the company are targeting. We can help you identify the best strategies to do so.
+
+## Keep plenty of breathing room
+
+We have found the team to be most effective when teammates keep plenty of breathing room: the team is more relaxed, has room for creativity, the quality of our work improves, and we end up making faster overall progress towards roadmap outcomes. Breathing room or slack time is traditionally hard to quantify, but as a ballpark idea try to cap roadmap commitments to 50-60% of your total bandwidth. We trust you with how you employ the rest of your bandwidth.
```

Comment:
Suggest a tighter definition/preface of what "breathing room" is or means in the first sentence before describing the benefits. Maybe it's just a question of swapping a variant of the 2nd sentence.

> Breathing room, or slack time, means leaving a portion of your total work schedule unallocated to specific tasks, which you can tap into  `(Rijnard says: or whatever definition we feel comfortable with)`. This portion can be difficult to quantify, but a ballpark [...] bandwidth per week `(Rijnard says: or whatever granularity of time. Worth mentioning because-at an extreme, slack time doesn't apply to spending 50% slack in the first part of 6 months and then 50% in the latter).` [This might be more than you're used to when planning work. We like plenty of breathing room because we've found the team to be more relaxed, ...]

Running out of steam finessing here but maybe that helps

Reply:
```suggestion
Breathing room, or slack time, means leaving a portion of your total work schedule unallocated to specific tasks. This portion is traditionally hard to quantify, but you should aim for a ballpark 50-60% of your total bandwidth. We trust you with how you employ this breathing room. We have found the team to be most effective when teammates keep plenty of breathing room: the team is more relaxed, has room for creativity, the quality of our work improves, and we end up making faster overall progress towards roadmap outcomes. 
```

This is what I came up with. Lmk if you think it still needs iteration!